Patches/ADE Advanced Turrets/Buildings_Security.xml

```diff
@@ -0,0 +1,184 @@
+				<xpath>/Defs/ThingDef[
+					defName="ADE_Turret_HMGT" or
+					defName="ADE_Turret_HST" or
+					defName="ADE_Turret_GLT"
+				]/fillPercent</xpath>
+				<value>
+					<fillPercent>0.85</fillPercent>
+				</value>
+			</li>
+			<!-- adding power requirement -->
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[
+					defName="ADE_Turret_HMGT" or
+					defName="ADE_Turret_HST" or
+					defName="ADE_Turret_GLT"
+				]/comps</xpath>
+				<value>
+				<li Class = "CompProperties_Power">
+			    <compClass>CompPowerTrader</compClass>
+				<basePowerConsumption>350</basePowerConsumption>
+				</li>
+				</value>
+			</li>		
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[
+					defName="ADE_Turret_HMGT" or
+					defName="ADE_Turret_HST" or
+					defName="ADE_Turret_GLT"
+				]/statBases/ShootingAccuracyTurret</xpath>
+				<value>
+					<AimingAccuracy>1</AimingAccuracy>
+					<ShootingAccuracyTurret>1</ShootingAccuracyTurret>
+				</value>
+			</li>
+			<!-- ========== HMG  Turret ========== -->
+
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+				<defName>ADE_Gun_HMGT</defName>
+				<statBases>
+					<RangedWeapon_Cooldown>0.18</RangedWeapon_Cooldown>
+					<SightsEfficiency>1</SightsEfficiency>
+					<ShotSpread>0.02</ShotSpread>
+					<SwayFactor>1.13</SwayFactor>
+					<Bulk>37.08</Bulk>
+				</statBases>
+				<Properties>
+					<recoilAmount>1.09</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_762x51mmNATO_FMJ</defaultProjectile>
+					<warmupTime>1.1</warmupTime>
+					<range>54</range>
+					<ticksBetweenBurstShots>3</ticksBetweenBurstShots>
+					<burstShotCount>20</burstShotCount>
+					<soundCast>Shot_AssaultRifle</soundCast>
+					<soundCastTail>GunTail_Medium</soundCastTail>
+					<muzzleFlashScale>9</muzzleFlashScale>
+					<recoilPattern>Mounted</recoilPattern>
+				</Properties>
+				<AmmoUser>
+					<magazineSize>400</magazineSize>
+					<reloadTime>15.6</reloadTime>
+					<ammoSet>AmmoSet_762x51mmNATO</ammoSet>
+				</AmmoUser>
+				<FireModes>
+					<aiUseBurstMode>FALSE</aiUseBurstMode>
+					<aiAimMode>AimedShot</aiAimMode>
+					<noSnapshot>true</noSnapshot>
+					<noSingleShot>true</noSingleShot>					
+				</FireModes>				
+			</li>
+
+			<!-- ========== Grenade Launcher Turret ========== -->
+
+			<li  Class="CombatExtended.PatchOperationMakeGunCECompatible">
+				<defName>ADE_Gun_GLT</defName>
+				<statBases>
+					<RangedWeapon_Cooldown>0.35</RangedWeapon_Cooldown>
+					<SightsEfficiency>1.00</SightsEfficiency>
+					<ShotSpread>0.14</ShotSpread>
+					<SwayFactor>2.02</SwayFactor>
+					<Bulk>10.66</Bulk>
+				</statBases>
+				<Properties>
+					<recoilAmount>0.34</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_40x46mmGrenade_HE</defaultProjectile>
```

Comment:
A minor quibble, but 40x53mm would be a bit more suited for this application. It's largely the same projectile, just higher velocity intended to be fired from vehicles.